mpicbg/src/main/java/mpicbg/models/InterpolatedAffineModel2D.java

```diff
@@ -72,6 +71,13 @@ public void set( final InterpolatedAffineModel2D< A, B > m )
 			affine.set( ( ( InterpolatedAffineModel2D< A, B > ) m ).affine );
```

Comment:
Both, the type test and the type cast seem unnecessary.  Please remove and replace with
```java
affine.set( m.affine );
```

Reply:
I didn't change this part so it's original code, but it does look redundant -- will change it while I'm on it